addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -1253,6 +1255,17 @@ registerModel({
         logs: attr({
             default: {},
         }),
+        /**
+         * Call all sessions for which no peerConnection is established at
+         * a regular interval to try to recover any connection that failed
+         * to start.
+         *
+         * This is distinct from this.recoverConnection which tries to restores
+         * connection that were established but failed or timed out.
+         */
+        missingRecoverInterval: attr({
```

Comment:
I find the "missing" name to be misleading, and it's strange to talk about a recover for something that hasn't happened yet.

The idea is probably in the doc, maybe "establish", or even just "connect"?